rolling_update.yml

```diff
@@ -59,7 +59,7 @@
       until: result.rc == 0
       retries: 5
       delay: 10
-      delegate_to: 127.0.0.1
+      delegate_to: "{{ groups.mons[0] }}"
```

Comment:
did you actually test this statement? can you confirm that it works?
I had so much trouble getting this work properly, that's why I ended up with this stupid with_items :)